pkg/typeutil/clone_test.go

```diff
@@ -0,0 +1,34 @@
+package typeutil
+
+import (
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_DeepClone(t *testing.T) {
+	re := assert.New(t)
+	src := &metapb.Region{Id: 1}
```

Comment:
BTW, would you like to add a test for a more complex struct, e.g., include more different types of fields?

Reply:
how about StoreHeartbeatRequest?